portals/developer-portal/samples/layouts/green-theme/styles/main.css

```
@@ -0,0 +1,507 @@
+}
+
+.common-btn-outlined:active {
+  background-color: var(--primary-light-color);
+  color: var(--light-text-color);
+}
+
+.common-btn-outlined:focus {
+  outline: none;
+}
+
+.common-btn-outlined disabled,
```

Stylelint (17.12.0)

[error] Unknown type selector "disabled" (selector-type-no-unknown)

(selector-type-no-unknown)